src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -219,9 +219,10 @@ private function getProductToolbarButtons(): array
         $toolbarButtons = [];
 
         $toolbarButtons['add'] = [
-            'href' => $this->generateUrl('admin_product_new'),
-            'desc' => $this->trans('New product', 'Admin.Actions'),
+            'href' => $this->generateUrl('admin_products_v2_create'),
+            'desc' => $this->trans('New product on experimental page', 'Admin.Actions'),
```

Comment:
@jolelievre, if this targets v8, do we need to call it `experimental` if the goal is for the new product page to be completed before release?

Reply:
Yep exactly, but we are still under development so we keep this wording, for now, to be able to tell both buttons apart When the new version will be finished we will remove the former controller (along with form types and templates) and update the wording appropriately
But for now it's convenient to have access to both pages